[Access] Remove instructions that required jq and lokey (#4077)
cloudflare/cloudflare-docs: content/cloudflare-one/identity/users/validating-json.md

## Manual verification

1.  Go to [jwt.io](https://jwt.io/).

1.  Select the RS256 algorithm.

> 1.  Paste the JWT into the field on the left. Get the `kid` value of the header box on the decoded column.

Reviewer: ```suggestion
1.  Paste the JWT into the **Encoded** field on the left. 

1. Note the `kid` value within the **Header** box on the right.
```

> 1.  Get your public key by going to `https://<your team domain>/cdn-cgi/access/certs` and copy the cert value of the block in `public_certs` with the `kid` value you found in the previous step.

Reviewer: ```suggestion
1.  Get your public key:
    1. Go to `https://<your team domain>/cdn-cgi/access/certs`.
    2. Under `public_certs`, locate the entry with the `kid` value you found in Step #3.
    3. Copy the `cert` value.
```

> 1.  Enter the public key in the **Public Key** field.

Reviewer: ```suggestion
1.  In the **Verify Signature** box, paste the `cert` value into the **Public Key** field.
```
